packages/server/src/cli.ts

```diff
@@ -235,6 +235,9 @@ const bold = (text: string | number): string | number => {
 		} : undefined,
 	});
 
+	if (options.noAuth && options.host === "0.0.0.0"|| options.allowHttp && options.host === "0.0.0.0") {
+		logger.warn("0.0.0.0 is accessible to any device on your network");
```

Comment:
Nit: maybe we can elaborate this more. Here's a suggestion
```suggestion
		logger.warn("0.0.0.0 is accessible to every device on the network your connected to. Make sure you set up authentication to prevent unauthorized access.");
```